src/main/java/ddingdong/ddingdongBE/file/dto/ExcelClubMemberDto.java

```diff
@@ -70,30 +64,31 @@ public static ClubMemberDto fromExcelRow(Row row) {
                 }
             } else if (cell.getCellType() == CellType.NUMERIC) {
```

Comment:
```suggestion
                }
                continue;
            }
       if (cell.getCellType() == CellType.NUMERIC) {
```
early return 사용해주는 것이 가독성이 더 괜찮아 보입니다!